mono/utils/networking-posix.c

```diff
@@ -21,6 +21,9 @@
 #include <sys/socket.h>
 #endif
 #ifdef HAVE_NET_IF_H
+#ifdef (__OpenBSD__)
+#include <sys/socket.h>
```

Comment:
There is already a sys/socket.h include above, isn't HAVE_SYS_SOCKET_H defined on openbsd ?

Reply:
Yes I'm pretty convinced this is just a bad merge. The older Mono code did not include sys/socket.h here (or nearby).